.github/workflows/validate_linux_rt.yml

```diff
@@ -0,0 +1,34 @@
+name: Validate Linux RT Feed
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  validate_linux_rt:
+    name: Validate Linux RT Feed
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Setup python3
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install build dependencies
+      run: python -m pip install -r python_build_requirements.txt
+
+    - name: Get previous tag
+      id: previoustag
+      run: >-
+        echo "::set-output name=tag::$(
+          git describe --abbrev=0 --tags ${{github.ref_name}}^
+        )"
+
+    - name: Update Linux RT's grpc-device dependency to ${{ github.ref_name }}?
```

Comment:
Is this aspirational? We're not actually updating here right? Maybe "Test update"

Reply:
Is someone going to be monitoring this script? Could we have this action make an AzDO work item in the backlog automatically?